test/test_jit.py

```diff
@@ -13455,8 +13455,11 @@ def checkTracerWarning(self, *args, **kwargs):
         with warnings.catch_warnings(record=True) as warns:
             torch.jit.trace(*args, **kwargs)
         self.assertGreater(len(warns), 0)
+        found_warning = False
         for warn in warns:
-            self.assertIn("cause the trace to be incorrect", str(warn.message))
+            if "cause the trace to be incorrect" in str(warn.message):
```

Comment:
can't you combine all of this into one line?

Reply:
Sure